init.lua

```diff
@@ -1,16 +1,5 @@
--- huge is over 100KB
-local is_huge_file = function(bufnr)
```

Comment:
this works only for `argv[1]` literally

but you can work with 1000 buffers and accidentally open a bigfile that will be 1001th buf. 

My approach gives special filetype to such files and applied local options (improve list and add what you consider useful)

it's also possible to have double types like `json.bigfile` and gracefully handle them.